core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message.groovy

```diff
@@ -0,0 +1,38 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2019 CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package jenkins.security.ResourceDomainRecommendation
+
+def f = namespace(lib.FormTagLib)
+
+dl {
+    div(class: "alert alert-info") {
```

Comment:
Just to be sure, the `alert-info` is not recognized by the default CSS theme and result in:
<details>

<summary>Screenshot of warnings</summary>

![Screenshot_2019-10-03_120913_001](https://user-images.githubusercontent.com/2662497/66118476-bd1bf980-e5d6-11e9-82d8-1daa22bb811b.png)


</details>

Due to the potential security issue related to the lack of CSP, I would advice to use `alert alert-warning` there.

Reply:
It's a long-standing bug in the popup, the full page has it (or rather, I think that warnings are colored in the popup is a bug, don't remember). The background is blue.

This isn't a warning, so it's not colored as such.